content/docs/csidriver/upgradation/drivers/isilon.md

```diff
@@ -8,18 +8,18 @@ Description: Upgrade PowerScale CSI driver
 ---
 You can upgrade the CSI Driver for Dell EMC PowerScale using Helm or Dell CSI Operator.
 
-## Upgrade Driver from version 1.6.0 to 2.0.0
+## Upgrade Driver from version 2.0.0 to 2.1.0
 
 **Note:** While upgrading the driver via helm, controllerCount variable in myvalues.yaml can be at most one less than the number of worker nodes.
 
 **Steps**
-1. Verify that all pre-requisites to install CSI Driver for Dell EMC PowerScale version 2.0.0 are fulfilled. Note that change in secret format should be taken care.
+1. Verify that all pre-requisites to install CSI Driver for Dell EMC PowerScale version 2.1.0 are fulfilled. Note that change in secret format should be taken care.
```

Comment:
Verify that all pre-requisites to install CSI Driver for Dell EMC PowerScale version 2.1.0 are fulfilled. Note that change in secret format should be implemented.

Reply:
@rsedlock1958 : Addressed